linux_os/guide/system/selinux/grub2_enable_selinux/rule.yml

```diff
@@ -20,7 +20,6 @@ severity: medium
 identifiers:
     cce@rhel7: CCE-26961-3
     cce@rhel8: CCE-80827-9
-    cce@rhcos4: CCE-82666-9
```

Comment:
Was this rule ever shipped in a release? If yes, I don't think it makes sense to remove the CCE even if it's not part of a profile anymore.

Reply:
OK, I'll change it.